tests/net/ieee802154/l2/src/ieee802154_test.c

```diff
@@ -983,6 +987,7 @@ ZTEST(ieee802154_l2, test_parsing_ack_pkt)
 	zassert_true(ret, "ACK parsed");
 }
 
+#ifdef CONFIG_NET_SOCKETS
```

Comment:
I wonder whether it'd make sense to define a separate `ZTEST_SUITE()` for raw socket tests?

Reply:
Gotcha, will do.

Reply:
Fixed.